pkg/maps/metricsmap/metricsmap.go

```diff
@@ -48,19 +48,23 @@ const (
 	// of 2**16 (2 uint8) to 2**10 (1 uint8 + 2 bits).
 	MaxEntries = 1024
 	// dirIngress and dirEgress values should match with
-	// METRIC_INGRESS and METRIC_EGRESS in bpf/lib/common.h
+	// METRIC_INGRESS, METRIC_EGRESS and METRIC_SERVICE
+	// in bpf/lib/common.h
+	dirEgress  = 0
 	dirIngress = 1
-	dirEgress  = 2
-	dirUnknown = 0
+	dirService = 2
+
+	dirUnknown = 255
```

Comment:
FWIW the directionality here used 0 as the "unknown" value in case there were any paths that logged metrics if the value was left uninitialized / as zero. I'm not sure if there's actually any paths left that would use that value for the direction.

If unknown isn't 0, I think there's no point in the unknown value. I don't think any path in the datapath will accidentally set the value to 255.